pkg/interface/chat/src/js/components/sidebar.js

```diff
@@ -145,11 +139,11 @@ export class Sidebar extends Component {
 
         return (
           <SidebarItem
-            key={obj.cir}
+            key={obj.cir + '/' + obj.wen}
```

Comment:
This feels like a hack, but it seems like it's [the way](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key) to solve the problem of having an interval update the timestamp state every minute, but also update whenever the wen-prop changes.

Reply:
I like this approach, actually! Very clean.